### PR TITLE
Add links to full size subjects in talk

### LINKED ISSUE #979: Adding a magnifier feature from Talk 2.0 into panoptes Talk
future feature request - I've seen a few request on P4T Talk to have the magnifier that exists on P4 Talk on the new panoptes talk. I don't think the magnifier is implemented in all of Talk 2.0 but for some projects like P4. 

Basically you moused over the subject and a zoom in window showed up that could move around the image to see different parts at higher resolution 

Since you can't see my mouse in a screenshot, I faked a mouse pointer in the image

screen shot 2015-07-06 at 9 26 06 am

### PR DESCRIPTION
**Pull request by aweiksnar**
- Adds links to full res subject images in discussions and subject page
- Closes #979

### CODE REVIEW

File: app/components/subject-viewer.cjsx [hunk 1]
```diff
@@ -103,6 +104,10 @@ module.exports = React.createClass
               <FavoritesButton project={@props.project} subject={@props.subject} user={@props.user} />
               <CollectionsManagerIcon project={@props.project} subject={@props.subject} user={@props.user} />
             </span>}
+          {if type is 'image' and @props.linkToFullImage
+            <a href={src} title="Subject Image" >
```
Comment (chrissnyder) [inline app/components/subject-viewer.cjsx]: Extra space before closing >

**Comment by chrissnyder** [comment]
Thoughts on having it open in a new window? It'd be the only action button below the subject that moves the user somewhere else.

**Comment by chrissnyder** [comment]
Other than that, LGTM

File: app/components/subject-viewer.cjsx (hunk 1 above)
Comment (aweiksnar) [inline app/components/subject-viewer.cjsx]: Good catch just updated

**Comment by aweiksnar** [comment]
Yeah agree that new window is better :+1: Just updated

**Comment by chrissnyder** [comment]
:shipit:

**Pull request merged**